css/styles.css

```diff
@@ -586,16 +578,113 @@ footer {
   flex-direction: row;
   justify-content: space-around;
   width: 200px;
-  list-style: none;
   margin: 0;
   padding: 0;
   align-items: center;
 }
 
-#hamburguerLink ul {
-  list-style: none;
+/* Details Popup */
+.popupContent {
+  display: none;
+  position: fixed;
+  width: 90%;
+  height: 100%;
+  top: 5%;
+  left: 5%;
+  right: 5%;
+  z-index: 100;
+}
+
+.detailsContainer {
```

Comment:
- [ ] Kindly adjust the styling of the modal to look like the Figma design the background is blurred this will help users concentrate more on the content in the modal and not background 👍🏿 

- [ ] Kindly use the same padding and margin to align contents in desktop and mobile, this will make your design look like the original 👍🏿 

- [ ] It would be better for the whole page to not be scrollable when the modal is open, only the modal should be scrollable 👍🏿 


 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/69638013/157589923-d7275b6d-627e-4f37-811b-827c0b79acb9.png)  | ![image](https://user-images.githubusercontent.com/69638013/157589731-d5a4fe74-b645-4200-be46-79c3f33dc310.png)